account_bank_statement_import_save_file/__init__.py

```diff
@@ -1,6 +1 @@
-# -*- coding: utf-8 -*-
-# © 2015 Therp BV (<http://therp.nl>).
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
You can keep the license in these files

Reply:
No nitpicking please :)

Reply:
None intended :) that was just a comment, not a change request ;)